app/private_publisher.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+#   coding: UTF-8
+
+from app.queue_publisher import QueuePublisher
+
+from cryptography.fernet import Fernet
+
+
+class PrivatePublisher(QueuePublisher):
+
+    @staticmethod
+    def encrypt(message, secret):
+        """
+        Message may be a string or bytes.
+        Secret key must be 32 url-safe base64-encoded bytes.
+
```

Comment:
Yes, they should be consistent. Which to use depends on house style:
[Docstring Conventions](https://www.python.org/dev/peps/pep-0257/).

I've gone with the blank line at the end because to me it looks nicer ;-)